packages/patternfly-4/react-core/src/components/Form/FormGroup.tsx

```diff
@@ -16,6 +16,8 @@ export interface FormGroupProps extends Omit<React.HTMLProps<HTMLDivElement>, 'l
   isRequired?: boolean;
   /** Sets the FormGroup isValid. */
   isValid?: boolean;
+  /** Sets the FormGroup validated. */
+  validated?: boolean;
```

Comment:
should we call this isValidated to match our other boolean modifier props?

Reply:
I was looking at this again.  I think having the 2 props isValid and validated may be a little confusing.  I know to make them one prop would be a breaking change.  Could we maybe make validated into an enum?  If the enum is used, isValid can be ignored. We can then deprecate isValid when we have a breaking change release.
`ValidatedOptions {
  success = 'success',
  error = 'error',
  default = 'default'
}`
 the modifiers would be applied if not default `validated !== ValidatedOptions.default`

Reply:
@tlabaj is this something that could be done all together for the next breaking change release? That is, leave it this way for now and change both `isValid` and `validated` to an enum value as a follow-on?

Reply:
It could.  I just worry that the two props could be confusing.  I wonder if we should at least make the comments more clear as to what the prop actually does.  To me isValid could implies that it was validated and it might not be clear you need to also apply validated.